xla/service/service.cc

```diff
@@ -748,7 +748,8 @@ absl::StatusOr<std::unique_ptr<Executable>> Service::BuildExecutable(
       module_proto.name());
 
   tsl::profiler::ScopedAnnotation annotation{[&] {
-    return absl::StrCat("XlaCompile:#module=", module_proto.name(), "#");
+    // module's unique_id is not available yet
```

Comment:
This is a bit annoying, because it means not all `XlaCompile` annotations look the same as each other, but it looks messy to predict what the unique_id will be on this line.